config/secrets.rb

```diff
@@ -0,0 +1,14 @@
+default: &default
```

Comment:
Fixed in https://github.com/codefordenver/fresh-food-connect-api/pull/45/commits/bd95cc0c9f14f8625eabdace8638de1b51aca45e